index.html

```diff
@@ -0,0 +1,128 @@
+            <li><a class="nav-link" href="#Hello_world">Hello world</a></li>
+            <li><a class="nav-link" href="#Variables">Variables</a></li>
+          </ul>
+        </div>
+      </nav>
+      <main id="main-doc">
+        <section id="Introduction" class="main-section">
+          <header>Introduction</header>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+        </section>
+        <section id="what_you_should_already_know" class="main-section">
+          <header>what you should already know</header>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+        </section>
+        <section id="JavaScript_and_Java" class="main-section">
+          <header>JavaScript and Java</header>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Neque,
+            necessitatibus!
+          </p>
+        </section>
+        <section id="Hello_world" class="main-section">
+          <header>Hello world</header>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+          </p>
+          <ul>
+            <li>Selam</li>
+            <li>Hello</li>
+            <li>Guten Morgon</li>
+            <li>Merhaba</li>
+            <li>Kaifa</li>
+          </ul>
+        </section>
+        <section id="Variables" class="main-section">
+          <header>Variables</header>
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+            Necessitatibus veniam dolores incidunt. Beatae molestias labore
+            perferendis facere? Libero, aut aperiam?
+            <code></code>
+            <code></code>
+            <code></code>
+            <code></code>
+            <code></code>
```

Comment:
- [ ] Please remove unused '<code>' tags because it makes your code cleaner and easy to read.